tests/legacy/unit/plugins/v2/test_colcon.py

```diff
@@ -182,6 +182,7 @@ class Options:
         "colcon build "
         '--base-paths "${SNAPCRAFT_PART_SRC_WORK}" --build-base "${SNAPCRAFT_PART_BUILD}" '
         '--merge-install --install-base "${SNAPCRAFT_PART_INSTALL}"/opt/ros/snap '
+        '--cmake-args -DCMAKE_BUILD_TYPE=Release '
```

Comment:
I might be missing something but here the legacy test is changed while the [legacy plugin](https://github.com/snapcore/snapcraft/blob/main/snapcraft_legacy/plugins/v2/colcon.py) is not.